code/script.js

```diff
@@ -0,0 +1,79 @@
+const USER = 'Groovedharry'
+const REPOS_URL = `https://api.github.com/users/${USER}/repos`;
+const USER_API = `https://api.github.com/users/${USER}`;
+const projectsContainerXXYT = document.getElementById('profileInfo');
+const projectsContainer = document.querySelector("#projects");
+
+
+const getUser = () => {
+    fetch(USER_API)
+        .then((response) => response.json())
+        .then((data) => {
+            profile.info.innerHTML += `
```

Comment:
I would guess that this bit doesnt work since you havent targeted the correct id. you have used profile.info.innerhtml. Try to change that to profileInfo.innerHTML instead.